extensions/SemanticMediaWiki/DefaultSettings.php

```diff
@@ -196,32 +196,32 @@
 # __HIDEFACTBOX__ can be used to control Factbox display for individual pages.
 # Other options for this setting include:
 ##
-// $GLOBALS['smwgShowFactbox'] = SMW_FACTBOX_NONEMPTY; # show only those factboxes that have some content
-// $GLOBALS['smwgShowFactbox'] = SMW_FACTBOX_SPECIAL # show only if special properties were set
-$GLOBALS['smwgShowFactbox'] = SMW_FACTBOX_HIDDEN; # hide always
-// $GLOBALS['smwgShowFactbox'] = SMW_FACTBOX_SHOWN;  # show always, buggy and not recommended
+// $smwDefaults['smwgShowFactbox'] = SMW_FACTBOX_NONEMPTY; # show only those factboxes that have some content
+// $smwDefaults['smwgShowFactbox'] = SMW_FACTBOX_SPECIAL # show only if special properties were set
+$smwDefaults['smwgShowFactbox'] = SMW_FACTBOX_HIDDEN; # hide always
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)